Automattic/wp-calypso
# Signup: Automatically login user after signup is complete

Until now, when a user signs up and reaches the processing screen, they were not logged in until they explicitly clicked the Continue button.

This PR aims to fix that, by automatically logging in the user when they finish the signup flow.

This is still a work in progress.

To test:
1. Checkout branch or use Calypso.live link
2. Stat Signup in main flow from `/start/`
3. Go through Signup, but don't click the `Continue` button at the end.
4. Reload the page
5. You should be logged in WordPress.com / Calypso

cc @jblz @lsinger @meremagee @michaeldcain
